site-scrapers/TuftsMC/index.js

```diff
@@ -0,0 +1,135 @@
+const https = require("https");
+const file = require("../../lib/file");
+const fetch = require("node-fetch");
+const { site, requestInit } = require("./config");
+
+module.exports = async function GetAvailableAppointments(
+    browser,
+    pageService = defaultPageService()
+) {
+    console.log(`${site.name} starting.`);
+    const webData = await ScrapeWebsiteData(browser, pageService);
+    console.log(`${site.name} done.`);
+    return {
+        ...site,
+        ...webData,
+        timestamp: new Date(),
+    };
+};
+
+function defaultPageService() {
+    return {
+        async fetchAvailability() {
+            return fetchAvailability();
+        },
+    };
+}
+
+async function ScrapeWebsiteData(browser, fetchService) {
+    // Initialize results to no availability
+    const results = {
+        availability: {},
+        hasAvailability: false,
+    };
+
+    /* Note:
+        site has next button if 'button[data-test="desktop-controls-right-arrow"]'
+        is not disabled
+    */
+
+    const response = await fetchService.fetchAvailability();
+
+    results.availability = {
+        ...results.availability,
+        ...parseResponse(response),
+    };
+    // }
+
+    results.hasAvailability = !!Object.keys(results.availability).length;
+
+    if (process.env.DEVELOPMENT) {
+        file.write(
+            `${process.cwd()}/out.json`,
+            `${JSON.stringify(results, null, "   ")}`
+        );
+    }
+
+    return results;
+}
+
+async function fetchAvailability() {
+    let fetchResponse = await fetch("https://api.zocdoc.com/directory/v2/gql", {
+        headers: {
```

Comment:
are these the same as the headers in config.js?

Reply:
Yes, it's remnant of an unfinished refactor to a constant called `requestInit` in `config.js`. But it disappears in the inclusion of Tufts MC in the ZocDoc scraper.